.github/workflows/units-tests-dispatch.yml

```diff
@@ -0,0 +1,45 @@
+name: Runs unit tests for a given ref
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'GitHub ref to test.'
+        required: false
+        default: 'master'
+        type: string
+      nodeVersion:
```

Comment:
can we please simplify this and only use 18.x here?

Reply:
Sure